### ISSUE TITLE
TypeError: unsupported operand type(s) for +=: 'Decimal' and 'NoneType'

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2125]
_Imported from trac ticket 2125.
Created by **sal**
Opened in trac: Feb 6, 2013, 1:11:15 PM
Last modified in trac: Feb 6, 2013, 2:22:56 PM_
***
_by sal:_
```
[Wed Feb 06 11:10:55 2013] [error] [client 208.115.5.118] File '/home/rarch/trunk/src/appserver/pylotengine/model/objects.py', line 17980 in process_financebalancing
[Wed Feb 06 11:10:55 2013] [error] [client 208.115.5.118]   bank[o]['credit'] += transaction['amount']
[Wed Feb 06 11:10:55 2013] [error] [client 208.115.5.118] TypeError: unsupported operand type(s) for +=: 'Decimal' and 'NoneType'
```

**Comment by kentbower** [comment]
_**kbower** trac comment on Feb 6, 2013, 1:15:00 PM:_

`transaction['amount']` is None for remove financing

**Comment by kentbower** [comment]
_**rneville** trac comment on Feb 6, 2013, 1:15:45 PM:_

Ticket #2122 has been marked as a duplicate of this ticket.

**Comment by kentbower** [comment]
_**kbower** trac comment on Feb 6, 2013, 2:22:56 PM:_

(Changeset [[3654]](../commit/adf44665c7cb6d5c362803b54f761721b31fbadc))
* fixes #2125 (TypeError: unsupported operand type(s) for +=: 'Decimal' and 'NoneType' when removing financing with split order)

**Comment by kentbower** [comment]
_**kbower** trac status change on Feb 6, 2013, 2:22:56 PM:_ **closed**
